### PR TITLE
rscript: add page

### PR DESCRIPTION
**Pull request by jhrcook**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):** 4.1.1

### CODE REVIEW

**Comment by jhrcook** [comment]
If accepted, can you please add the label `hacktoberfest-accepted` so I can get credit for [Hacktoberfest](https://hacktoberfest.digitalocean.com)?

File: pages/common/rscript.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start an Rscript in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
+
+`Rscript --vanilla {{file.R}}`
+
+- Check R version:
+
+`Rscript --version`
+
+- Execute one or more R expressions and then exit:
```
Comment (pixelcmtd) [inline pages/common/rscript.md:18]: It's kinda obvious that a program exits.

```suggestion
- Execute one or more R expressions:
```

File: pages/common/rscript.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# R
+
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an R script:
+
+`Rscript {{file.R}}`
+
+- Start an Rscript in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
```
Comment (pixelcmtd) [inline pages/common/rscript.md:10]: Could you please decide to call it `R script` or `Rscript`, or is there any difference?

File: pages/common/rscript.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# R
```
Comment (navarroaxel) [inline pages/common/rscript.md:1]:
```suggestion
# rscript
```

File: pages/common/rscript.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an R script:
+
+`Rscript {{file.R}}`
```
Comment (navarroaxel) [inline pages/common/rscript.md:8]: We use `path/to/file`
```suggestion
`Rscript {{path/to/file.R}}`
```

File: pages/common/rscript.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`Rscript --vanilla {{file.R}}`
+
+- Check R version:
+
+`Rscript --version`
+
+- Execute one or more R expressions and then exit:
+
+`Rscript -e {{expr1}} -e {{expr2}}`
```
Comment (navarroaxel) [inline pages/common/rscript.md:20]:
```suggestion
`Rscript -e {{expression1}} -e {{expression2}}`
```

File: pages/common/rscript.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`Rscript {{file.R}}`
+
+- Start an Rscript in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
+
+`Rscript --vanilla {{file.R}}`
+
+- Check R version:
+
+`Rscript --version`
```
Comment (navarroaxel) [inline pages/common/rscript.md:16]: this should be the last example because it's not very useful.

File: pages/common/rscript.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an R script:
+
+`Rscript {{file.R}}`
+
+- Start an Rscript in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
+
+`Rscript --vanilla {{file.R}}`
```
Comment (navarroaxel) [inline pages/common/rscript.md:12]:
```suggestion
`Rscript --vanilla {{path/to/file.R}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/rscript.md (hunk 2 above)
Comment (jhrcook) [reply pages/common/rscript.md:10]: Sorry for the confusion, I have made changes to clarify this.

File: pages/common/rscript.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Rscript
+
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an script:
+
+`Rscript {{path/to/file.R}}`
+
+- Run an script in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
```
Comment (CleanMachine1) [inline pages/common/rscript.md:10]:
```suggestion
- Run a script in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/rscript.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# Rscript
+
+> Run a script with the R programming language.
+> More information: <https://www.r-project.org>.
+
+- Run an script:
```
Comment (navarroaxel) [inline pages/common/rscript.md:6]:
```suggestion
- Run a script:
```

File: pages/common/rscript.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run a script in vanilla mode (i.e. a blank session that doesn't save the workspace at the end):
+
+`Rscript --vanilla {{path/to/file.R}}`
+
+- Execute one or more R expressions:
+
+`Rscript -e {{expression1}} -e {{expression2}}`
+
+- Check R version:
```
Comment (navarroaxel) [inline pages/common/rscript.md:18]:
```suggestion
- Display R version:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**